docs/src/main/sphinx/release/release-0.209.md

```diff
@@ -25,7 +25,7 @@
 - Add the {func}`convex_hull_agg` and {func}`geometry_union_agg` geospatial aggregation functions.
 - Add `(TYPE IO, FORMAT JSON)` option for {doc}`/sql/explain` that shows
   input tables with constraints and the output table in JSON format.
-- Add {doc}`/connector/kudu`.
+- Add Kudo connector.
```

Comment:
Kudo -> Kudu

Reply:
oops .. I had that same spelling mess up elsewhere and caught .. thank you @ebyhr